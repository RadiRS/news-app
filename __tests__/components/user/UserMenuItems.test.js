import 'react-native';
import 'jest-styled-components';
import React from 'react';
import renderer from 'react-test-renderer';
import UserMenuItem from '../../../src/components/user/UserMenuItem';

test('renders UserMenuItem component correctly', () => {
  const tree = renderer
    .create(<UserMenuItem icon="ios-add" title="" text="" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
