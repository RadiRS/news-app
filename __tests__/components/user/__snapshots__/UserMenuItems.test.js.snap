// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders UserMenuItem component correctly 1`] = `
<View
  style={
    Array [
      Object {
        "flexDirection": "row",
        "marginBottom": 15,
        "marginLeft": 0,
        "marginRight": 0,
        "marginTop": 15,
      },
    ]
  }
>
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
          "height": 32,
          "justifyContent": "center",
          "width": 32,
        },
      ]
    }
  >
    <Text
      allowFontScaling={false}
      style={
        Array [
          Object {
            "color": "#546bfb",
            "fontSize": 24,
          },
          undefined,
          Object {
            "fontFamily": "Ionicons",
            "fontStyle": "normal",
            "fontWeight": "normal",
          },
          Object {},
        ]
      }
    >
      ï„‚
    </Text>
  </View>
  <View
    style={
      Array [
        Object {
          "paddingLeft": 20,
        },
      ]
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#3c4560",
            "fontSize": 24,
            "fontWeight": "600",
          },
        ]
      }
    >
      
    </Text>
    <Text
      style={
        Array [
          Object {
            "color": "#3c4560",
            "fontWeight": "600",
            "marginTop": 5,
            "opacity": 0.6,
          },
        ]
      }
    >
      
    </Text>
  </View>
</View>
`;
